Core.Telemetry/Core.Telemetry.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+   <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>Microsoft.CloudMine.Core.Telemetry</RootNamespace>
+    <AssemblyName>Microsoft.CloudMine.Core.Telemetry</AssemblyName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Bond.Core.CSharp" Version="9.0.1" />
+    <PackageReference Include="Bond.CSharp" Version="9.0.1" />
+    <PackageReference Include="Bond.Runtime.CSharp" Version="9.0.1" />
```

Comment:
Please check the minimal set of dependencies needed. I would certainly hope that telemetry does not depend on Bond stuff (IFX might but not telemetry).